requirements.txt

```diff
@@ -1,7 +1,5 @@
 pyodbc
 six
-sqlalchemy
-#-e git+https://github.com/ODM2/geoalchemy.git@v0.7.4#egg=geoalchemy-0.7.4
 #shapely
```

Comment:
Might as well also remove the commented-out `shapely`. It's pointless or totally optional w/o geoalchemy.